components/list_item/list_item.vue

```diff
@@ -224,14 +243,15 @@ export default {
   border-radius: var(--size-300);
 }
 
-.dt-list-item--focusable:focus,
-.dt-list-item--focusable:focus-within,
 .dt-list-item--highlighted {
   background-color: hsla(var(--black-900-h), var(--black-900-s), var(--black-900-l), 0.05);
 }
 
-.dt-list-item--highlighted:active {
-  background-color: hsla(var(--black-900-h), var(--black-900-s), var(--black-900-l), 0.1);
+.dt-list-item--selected {
+  background-color: var(--purple-100);
+  .d-icon{
+    color: var(--purple-400);
```

Comment:
Remove color, deferring to inherited color.

Reply:
Since it will not have background color and the icon will have inherited color, I removed `.dt-list-item--selected` completely :)